docs/src/wiki/FAQ.md

```diff
@@ -0,0 +1,56 @@
+## FAQ
+
+- **What is Signer?**
+    
+    Signer is an app for an air-gapped device, it turns an offline device — usually a smartphone — into a secure hardware wallet. Signer offers you a way to securely generate, store, manage and use your blockchain credentials. 
+    
+- **How does an offline device communicate with the outside world?**
+    
+    Communication happens through QR codes that are scanned as an input into Signer to interact with your keys and then, in turn, it generates a response QR code on behalf of those keys that are meant to be scanned by the app you interact with. There are tried and true cryptographic algorithms that power these QR codes, as well as some smart engineering that make your dedicated device safe to use.
```

Comment:
> ...in turn, it generates...
Not clear what "it" is, probably we should chop this sentence in pieces.

> ...on behalf of those keys...
maybe we should just say that the message is signed by those keys and also emphasize that the secret keys are never exposed to external world?

Reply:
thanks, I've simplified that indeed weird sentence